src/modules/avformat/consumer_avformat.c

```diff
@@ -1896,8 +1896,26 @@ static void *consumer_thread( void *arg )
 					if ( mlt_properties_get_int( frame_properties, "rendered" ) )
 					{
 						AVFrame video_avframe;
+						int is_interlaced_chroma_correction = 0;
+
 						mlt_frame_get_image( frame, &image, &img_fmt, &img_width, &img_height, 0 );
 
+						// Interlaced 420 correction
+						if ( mlt_properties_get( frame_properties, "progressive" )
+						     && mlt_properties_get_int( frame_properties, "progressive" ) == 0
```

Comment:
No need for `mlt_properties_get()`. These two conditions should be simplified to
`!mlt_properties_get_int( frame_properties, "progressive" )`
since getting a property that does not exist as integer returns 0.

Reply:
I know that it will return 0, but I wanted to be sure, that this property exists. I don't want frame without this property considered as interlaced. May be property is just missed (or it is impossible?). But if you still want I will make changes.

Reply:
> I don't want frame without this property considered as interlaced.

But this is the way MLT works. The default is interlaced.

Reply:
I made changes. But I'm not sure should I press button "Resolve conversation" or something else. It is my first pull request on github.